### PR TITLE
fix #47

### LINKED ISSUE #47: About Page Suggestion
https://www.mozillascience.org/about

I have a suggestion for the "About" page, would be worth it to have a separate section for the MSL staff members at Mozilla instead of having it mixed with the community members.

### PR DESCRIPTION
**Pull request by mmhyamin**
1. node_modules, IDE, OS related files added in gitignore.
2. Staff list added.

### CODE REVIEW

**Comment by acabunoc** [comment]
Good point on adding a role for staff - that might be a better approach.

I'll take a closer look tomorrow, thanks for your work here :)

File: views/about.jade [hunk 1]
```diff
@@ -23,6 +23,9 @@ html
         div.pure-u-1.pure-u-md-3-4.spaced-columns
           p The Mozilla Science Lab is helping a global network of researchers, tool developers, librarians and publishers collaborate to further science on the web. We believe a community of peers working, learning and building together is needed to make research thrive on the open web.
 
+          h3 MEET OUR STAFF
+          p Here are the staff who are helping to advance science on the web.
```
Comment (acabunoc) [inline views/about.jade]: Can we change the wording here to @belkinsa's version? https://github.com/mozillascience/site/pull/55/files

Thanks!

File: routes/users.js [hunk 2]
```diff
@@ -228,7 +228,27 @@ module.exports = function() {
                              { location: regex } ] }, function(err, user){
         if (err) return console.error(err);
         res.json(user);
-      })    }
+      })
+    },
+    getStaffs : function(req, res, next){
+
+        User.find({
```
Comment (acabunoc) [inline routes/users.js]: I've added "staff" as a role in the database. Code below should work now! Thanks for the harcoded love in the meantime, though :heart: 

```
      User
        .find({role:"staff"})
        .select('-email -token')
        .exec(function (err, staff) {
          if (err) return console.error(err);
          res.json(staff);
      });
```

I added ` .select('-email -token')` for security :)

File: routes/users.js [hunk 3]
```diff
@@ -228,7 +228,27 @@ module.exports = function() {
                              { location: regex } ] }, function(err, user){
         if (err) return console.error(err);
         res.json(user);
-      })    }
+      })
+    },
+    getStaffs : function(req, res, next){
```
Comment (acabunoc) [inline routes/users.js]: Can we change the name to `getStaff` instead of `getStaffs`? It'll help me remember and maintain!

File: app.js [hunk 4]
```diff
@@ -281,6 +281,10 @@ app.post("/projects/:project/join", localQuery, projectRoutes.join);
 app.get("/api/users", userRoutes.getAll);
 app.post("/api/users/create", userRoutes.create);
 
+//start
+app.get("/api/staffs", userRoutes.getStaffs);
```
Comment (acabunoc) [inline app.js]: Change to: `/api/staff` instead of `/api/staffs`
Staff works as the plural. Thanks!

File: .gitignore [hunk 5]
```diff
@@ -1 +1,187 @@
-node_modules
+##################
```
Comment (acabunoc) [inline .gitignore]: Thanks for the comprehensive .gitignore!

**Comment by acabunoc** [comment]
Thank you so much for this @mmhyamin! I added a few comments inline, should be good to go once they're addressed. This is amazing work!

thank you

**Comment by Blahah** [comment]
Should be safe to do a find and replace `staffs` -> `staff` I think?

Great work turning this around so fast @mmhyamin!

**Comment by mmhyamin** [comment]
@Blahah yah headache of WebStorm. <3 WebStorm.

**Comment by acabunoc** [comment]
Amazing, @mmhyamin! :+1: 
Merging :smile:

**Pull request merged**